src/common/io_exerciser/Model.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "IoOp.h"
+
+#include <boost/asio/io_context.hpp>
+
+#include "librados/librados_asio.h"
+
+#include "include/interval_set.h"
+#include "global/global_init.h"
+#include "global/global_context.h"
+#include "common/Thread.h"
+
+/* Overview
+ *
+ * class Model
+ *   Virtual class. Models apply IoOps generated by an
+ *   IoSequence, they can choose how many I/Os to execute in
+ *   parallel and scale up the size of I/Os by the blocksize
+ *
+ */
+
+namespace ceph {
+  namespace io_exerciser {
+
+    class Model
+    {
+    protected:
+      int num_io;
```

Comment:
int num_io{0}, as you only have one ctor, and that's what it sets

Reply:
Will update. Is this just a readability improvement? I presume the compiled code that comes from this will still be the same?

Reply:
Not just readability. It also guarantees that if anyone ever adds a 2'nd ctor - that initialization would not be forgotten.
And yes - the resulting code is the same.